src/style.css

```
@@ -1 +1,12 @@
-/* styles go here */
+.form-1 {
+  display: block;
+  margin-top: 10px;
```

```suggestion
  margin-bottom: 10px; 
```
Add this class to each label